recipe/recipe.yaml

```diff
@@ -2,15 +2,18 @@ schema_version: 1
 
 context:
   name: xrootd
-  version: "5.8.1"
+  version: "5.8.2"
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
   url: https://github.com/xrootd/xrootd/archive/v${{ version }}.tar.gz
-  sha256: a8b4d6d8aa9cc2cba6e77a3f14deea9372de9be6fec2b1949ddf81bfb8aa4092
+  sha256: 3ceca2064b521523c9c4456767f10f3251bdcd819b7c55f1a47132067fa3394f
```

Comment:
The official tarballs with XRootD releases are (for the latest version, v5.8.4):
- https://xrootd.web.cern.ch/download/v5.8.4/xrootd-5.8.4.tar.gz
- https://github.com/xrootd/xrootd/releases/download/v5.8.4/xrootd-5.8.4.tar.gz

Please consider moving to one of the links above, rather than use the GitHub generated tarballs.

Reply:
Also, a more important question, do you want to be added as to the maintainers list for the feedstock? :)

Reply:
The official tarballs are the ones I generate myself with git archive locally from the tag and upload (see gen-tarball.sh script in the repo). The link you are using is for a dynamically generated tarball by GitHub, which has changed in the past, and although they are promising not to do it again, I wouldn't bet on it. See https://github.blog/changelog/2023-01-30-git-archive-checksums-may-change/

Sure, you can add me to the list and also ping me if anything needs my attention. Cheers,

Reply:
conda-forge keeps track of the checksum, so if the github tarball checksum changes, it will be noticed here. But this is only grabbed upon the initial time of the recipe, and any re-rendering that is being done -- but not during install.

Reply:
The suggestion of Guilherme is the correct approach, the lack of byte-for-byte stability in git archive is annoying (even if it has a good-ish reason...)

I'd suggest the GitHub URL as EOS web isn't the best behaved webserver...